lib/hardware/modules/div/iob_div_subshift_frac/iob_div_subshift_frac.py

```diff
@@ -176,26 +173,16 @@ def setup(py_params_dict):
             incr        = 1'b0;
             res_acc_nxt = res_acc + remainder_o;
             res_acc_en  = 1'b0;
-            pc_nxt      = pc + 1'b1;
-                """,
-            }
-        ],
-        "fsms": [
-            {
-                "verilog_code": """
-        idle:
-
-        #case 0
             if (!start_i) begin
                 pc_nxt = pc; 
             end 
 
-        #case 1
+      
             if (!done_o) begin
                 pc_nxt = pc;
             end
         
-        #default
+       
             begin 
```

Comment:
Since this snippet is now a 'comb' (no longer a py2 'fsm'), then the `case (pc)` line and the sates need to be implemented manually. Otherwise, the generated code will not be an fsm (will not be equal to the original [iob_div_subshift_frac.v](https://github.com/IObundle/iob-soc/blob/if_gen2/lib/hardware/modules/div/iob_div_subshift_frac/hardware/src/iob_div_subshift_frac.v#L83-L101)).